web/src/lib/components/listen-box.tsx

```diff
@@ -138,7 +138,9 @@ export default class ListenBox extends Component<Props, State> {
           <Icon type="x"/>Nope.</a>
       </div>
       <audio className="audio-box"
-        src={this.props.src}
+        // Only include the src attribute if the source is defined
+        // (empty src attributes are invalid)
+        {...this.props.src && {src: this.props.src}}
```

Comment:
Here is [some documentation](https://facebook.github.io/react/docs/jsx-in-depth.html#spread-attributes).